clang-tools-extra/docs/clang-tidy/checks/bugprone/unsafe-functions.rst

```diff
@@ -19,6 +19,9 @@ The check implements the following rules from the CERT C Coding Standard:
 Unsafe functions
 ----------------
 
+The following functions are reported if `ReportDefaultFunctions
+<unsafe-functions.html#cmdoption-arg-ReportDefaultFunctions>`_ is enabled.
+
```

Comment:
I hope I did it correctly, I'm not that familiar with RST syntax.

Reply:
Referring to the `.html` directly is definitely not good because rst can be lowered to different markup syntaxes.

You need to create (or check if `opt::` automatically does?) a referable entity, and then `:ref:` can be used to make the link clickable. So you will use the ID or "label" you allocated in the document instead of relying on the automatically generated, HTML-specific anchor.